### DISCUSSION TITLE
Need migration document for upgrading the Feast from 0.9 to latest version

### DISCUSSION THREAD

**Discussion by mayurikumari047** [feast-dev/feast#4345] [Q&A]
Hi, 
I see there is a link to get the migration document here in the page https://docs.feast.dev/project/feast-0.9-vs-feast-0.10+ . When I try to go to this link, it takes me to Google Docs request access page. I requested for the access couple of times but didn't receive. Please help.
Thanks,
Mayuri

**Comment by franciscojavierarceo** [comment]
It looks like this may no longer be available. I requested access as well. Are you currently on 0.9?

**Reply by mayurikumari047** [reply]
Yeah, I am currently on 0.9.

**Comment by tokoko** [comment]
We should probably remove that page from the docs altogether. It has been 3 years already and feast has changed considerably after 0.10

**Reply by mayurikumari047** [reply]
But, it could be beneficial for those who haven't upgraded yet and are planning to upgrade. If anyone else has this document. Please share. It would be of great help.

**Comment by franciscojavierarceo** [comment]
Feast 0.9 is no longer supported, @mayurikumari047 so I'm going to close this thread. Let us know if you encounter any challenges.
